globaldog.py

```diff
@@ -63,7 +65,7 @@ def handleErrors(datadogResponse):
 screenboard["board_title"] = "{} {}".format(
```

Comment:
```python
print("Sending updated screenboard to Datadog...")
```

The change to using "pint" appears to be a typo. Reverting back to "print" avoids potential runtime errors.